pegasus/sites.v3/code.org/public/dance-landing.haml

```diff
@@ -0,0 +1,33 @@
+%h1= I18n.t(:dance_main_heading)
+
+.dance-subheading
+  %h4= I18n.t(:dance_subheading)
+
+%link{href: "/css/starwars.css", rel: "stylesheet"}
```

Comment:
My extraction suggestion is to make a more general haml partial that has generic css, something like full-width-hoc-promo (or whatever you want to call it) that you parameterize and use for minecraft, starwars and dance.  Here you're still reusing starwars.css.  Is that your question?

Reply:
Yes, this was part of my question.  Initially, I imported styling from star wars.  Then I added a style sheet for dance-landing.  I created the generic shared-css file to address duplicate styles for the landing pages.

Reply:
I don't see that "generic shared-css" file in this PR - should I?